packages/Heading/stories/examples/Variations.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Rule, Story, Tagline } from "storybook/assets/styles/common.styles";
+import { HeadingStory } from "../Heading.stories.styles";
+import SetWidthExample from "./SetWidthExample";
+import ParagraphExample from "./ParagraphExample";
+import DisplayLevelExample from "./DisplayLevelExample";
+import IsLightExample from "./IsLightExample";
+import DividerExample from "./DividerExample";
+import UnderLineExample from "./UnderlineExample";
+import Heading from "../../src/Heading";
+
+export default function Variations() {
+  return (
+    <Story>
+      <Heading level={1} displayLevel={2} isLight>
+        Variations of Headings
+      </Heading>
+      <Rule />
+      <Tagline>Heading Levels</Tagline>
```

Comment:
I know we discussed using a new `<StoryHeading>` component, created in the `storyHelpers`, but if you want to save that for a separate task, that would actually make a lot of sense.